tasks/electron.js

```diff
@@ -4,7 +4,7 @@ const electron = require('electron');
 let subprocess;
 
 function startElectron(done) {
-  subprocess = spawn(electron, ['.'], {
+  subprocess = spawn(electron, ['.', '--no-sandbox'], {
```

Comment:
`--no-sandbox` is necessary because on Linux, if the `chrome-sandbox` binary shipped with Electron does not have the proper file system permission set (i.e `4755`), then Chromium won't be able to start. In production, `electron-builder` should be setting the right permissions on post install.